central/billingmetrics/csv_handler.go

```diff
@@ -0,0 +1,48 @@
+package billingmetrics
+
+import (
+	"net/http"
+	"time"
+
+	bmstore "github.com/stackrox/rox/central/billingmetrics/store"
+	"github.com/stackrox/rox/pkg/protoconv"
+)
+
+// CSVHandler returns an HTTP handler function that serves billing data as CSV.
+func CSVHandler() http.HandlerFunc {
+	store := bmstore.Singleton()
+	return func(w http.ResponseWriter, r *http.Request) {
+		if err := r.ParseForm(); err != nil {
+			w.WriteHeader(http.StatusBadRequest)
+		}
+		var err error
+		from := time.Unix(0, 0).UTC()
```

Comment:
I would suggest extracting this as a function outside of the handler. Then you can also use the real code in the test, instead of duplicating the `time.Parse`.

Reply:
Refactored. Please see #7136.